apps/web/src/ee/clerk/pages/ManageAccountPage.tsx

```diff
@@ -0,0 +1,149 @@
+import './ManageAccountPage.css';
+import { OrganizationProfile, UserProfile } from '@clerk/clerk-react';
+import { useColorScheme } from '@novu/design-system';
+import { Modal, Tabs } from '@mantine/core';
+import {
+  IconAdminPanelSettings,
+  IconCreditCard,
+  IconGroups,
+  IconLocalActivity,
+  IconManageAccounts,
+  IconRoomPreferences,
+} from '@novu/novui/icons';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ROUTES } from '../../../constants/routes';
+import { useFeatureFlag } from '../../../hooks/useFeatureFlag';
+import { BrandingPage } from '../../../pages/brand/BrandingPage';
+import { BillingPage } from '../../billing/pages/BillingPage';
+import { Title } from '@novu/novui';
+import { css } from '@novu/novui/css';
+
+const clerkComponentAppearance = {
+  elements: {
+    navbar: { display: 'none' },
+    navbarMobileMenuRow: { display: 'none !important' },
+    cardBox: {
+      display: 'block',
+      width: '100%',
+      height: '100%',
+      boxShadow: 'none',
+    },
+  },
+};
+
+export default function ManageAccountPage() {
+  const navigate = useNavigate();
+  const { tabValue } = useParams();
+  const isV2Enabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_V2_ENABLED);
+  const { colorScheme } = useColorScheme();
+  const headerColor = colorScheme === 'dark' ? 'rgb(255, 255, 255)' : 'rgb(33, 33, 38)';
+
+  return (
+    <Modal
+      opened
+      padding={8}
+      title={undefined}
+      onClose={() => {
+        navigate(ROUTES.WORKFLOWS);
+      }}
+      size="90%"
+    >
+      <>
+        <Tabs value={tabValue} onTabChange={(value) => navigate(`/manage-account/${value}`)} orientation="vertical">
+          <Tabs.List>
+            <Tabs.Tab
+              disabled
+              value="title"
+              className={css({
```

Comment:
I am confused with the addition of inline CSS. Mantine promotes css via React className but we also introduced a `ManageAccountPage.css` with BEM style classes. What was the reasoning behind this split?

Reply:
This is a workaround to make one of the tabs look like the title. The tabs themselves have styles (the look of all other buttons) but I needed to override this particular one to be specific. I think I could give it a specific class name and then target that from the css file, wdyt?

Reply:
That's an option. The addition of the .css file is a bit unexpected. If not mistaken in Mantine css in classname should be preferred.

Reply:
I've tried to move the definition to the .css file but it didn't work as expected. It will be another thing I will try to target with Joel